src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -308,10 +312,11 @@ private void UpdateMenuButtons()
         }
     }
 
-    private async Task ToggleTimestamp(bool hideTimestamp)
+    private async Task ToggleTimestampAsync(bool hideTimestamp)
     {
-        PageViewModel.HideTimestamp = hideTimestamp;
-        await this.AfterViewModelChangedAsync(_contentLayout, waitToApplyMobileChange: false);
+        await SessionStorage.SetAsync(HideTimestampStorageKey, new ConsoleLogTimestampState(HideTimestamp: hideTimestamp));
```

Comment:
Use browser storage instead of session storage so it persists between app runs. It should be like the persisted settings for the details view.